src/index.js

```diff
@@ -0,0 +1,3 @@
+import nabs from './nabs';
```

Comment:
I'd prefer to not re-organize the code in this PR as there's another PR pending for reorganization. Can you please stick with the original organization in this PR?